src/cli/repo/s3.go

```diff
@@ -109,22 +109,31 @@ func initS3Cmd(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
-	s, a, err := config.GetStorageAndAccount(ctx, false, s3Overrides())
+	cfg, err := config.GetConfigRepoDetails(ctx, false, s3Overrides())
 	if err != nil {
 		return Only(ctx, err)
 	}
 
-	s3Cfg, err := s.S3Config()
+	// SendStartCorsoEvent uses distict ID as tenant ID because repoID is still not generated
+	utils.SendStartCorsoEvent(
+		ctx,
+		cfg.Storage,
+		cfg.Account.ID(),
+		map[string]any{"command": "init repo"},
+		cfg.Account.ID(),
```

Comment:
shouldn't this be the repo ID?
```suggestion
		cfg.RepoID,
```

Reply:
For `repo init` command the repoID will be generated after few initialisation. And since we want this event to be the first thing happening, had to make this command as exception. And had to use Account ID as the unique ID here.

Reply:
Hmm.  Okay, lets move ahead with this.  I"ll talk to Georgi and see what value will work best for his end of things.  Thanks!